helm/thingsboard/templates/node.yaml

```diff
@@ -123,6 +123,8 @@ spec:
           {{- end }}
           - name: DEFAULT_INACTIVITY_TIMEOUT
             value: {{ .Values.node.state.defaultInactivityTimeoutInSec | default 600 | quote }}
+          - name: DEFAULT_STATE_CHECK_INTERVAL
+            value: {{ .Values.node.state.defaultStateCheckIntervalInSec | default 60 | quote }}
```

Comment:
Out of the scope of this PR, since it happens for more properties, but why do we have the default value duplicated both here and in values.yaml?

Reply:
In understood that in values.yaml was because we would be able to change them to test it in the e2e tests. And here simply to have them set in thingsboard(?)

Reply:
IMPO the right place (single point of truth) for the default values should be in values.yaml. WDYT?
anyway, I think that we are duplicating them as we are just duplicating the existing code for setting new variables.